src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/MatchScoreClient.kt

```diff
@@ -38,4 +41,24 @@ class MatchRecord(
   @JsonProperty("dob")
   val dateOfBirth: String? = "",
   val pnc: String? = "",
-)
+) {
+  companion object {
+    fun from(newRecord: Person): MatchRecord {
+      return MatchRecord(
+        firstName = newRecord.firstName,
+        lastname = newRecord.lastName,
+        dateOfBirth = newRecord.dateOfBirth?.toString(),
+        pnc = newRecord.references.firstOrNull { it.identifierType == IdentifierType.PNC }?.identifierValue,
```

Comment:
person.getPnc() would be more convenient - maybe an extension function would help